src/backends/tensorrt/tensorrtlib.cc

```diff
@@ -408,6 +426,9 @@ namespace dd
     nvinfer1::IHostMemory *n
         = _builder->buildSerializedNetwork(*network, *_builderc);
 
+    if (n == nullptr)
+      throw MLLibInternalException("Could not build model");
```

Comment:
Can we log slight more information about the model, most useful for debugging purposes.